tools/utils/__init__.py

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from .logger import get_logger
```

Comment:
are all of these scripts same with ORT? can we download it from ORT repo.
There always are some accusations of using the same code from ORT.

Reply:
Largely the same, but downloading ORT in order to use a couple of helper scripts feels like a really heavy dependency. If it was production code it would be worth sharing the code though.